src/web/modules/gateway_accounts/detail.njk

```diff
@@ -186,7 +193,32 @@
       })
       }}
     {% endif %}
+    {% if account.allow_moto === true %}
+      <form method="POST" action="/gateway_accounts/{{ gatewayAccountId }}/moto_payments">
+        {{ govukButton({
+          text: "Disable MOTO payments"
+          })
+        }}
+        <input type="hidden" name="_csrf" value="{{ csrf }}">
+      </form>
+    {% endif %}
+  </div>
+
+{% if account.allow_moto === false %}
+  <div>
+    <h1 class="govuk-heading-s">Enable MOTO payments</h1>
+    <p class="govuk-body">Please ensure the service has passed PCI requirements before enabling MOTO payments.</p>
+    <p class="govuk-body">MOTO payments allows payments to bypass security as there are no 3DS checks and billing addresses aren't required. This is inherently insecure.</p>
```

Comment:
Would it be good to make clear that this doesn't affect all payments created against the account, but allows them to include a flag that creates payments that would bypass these checks?